src/libfuncs/felt252.rs

```diff
@@ -345,6 +355,13 @@ mod test {
         u32_digits.try_into().unwrap()
     }
 
+    #[test]
+    fn felt252_factorial() {
+        let r = |n| run_program(&FELT252_FACTORIAL, "factorial", json!([null, 0, n]));
+
+        assert_eq!(r(f("5")), json!([null, 0, [0, [f("120")]]]));
+    }
+
```

Comment:
Same as with the other PR, I think we should move this into the integration tests.

Reply:
i'll move this in a following pr